dsl_test.go

```diff
@@ -29,14 +53,55 @@ func TestBuilder(t *testing.T) {
 			{"staging", "staging.api.com"},
 			{"production", "api.com"},
 		},
+		Paths: map[string]ogen.PathItem{
+			pathWithID: {
+				Description: "This is my first path",
+				Get: &ogen.Operation{
+					Tags:        []string{"default"},
+					Description: "Description for my path",
+					OperationID: "path-with-id",
+					Parameters: []ogen.Parameter{
+						{
+							Name:        "id",
+							In:          "path",
+							Description: "ID param in path",
+							Required:    true,
+							// TODO: Schema
+							// TODO: Required
+							// TODO: Deprecated
+							// TODO: Content
+							// TODO: Style
+							// TODO: Explode
+						},
+						{Ref: "#/components/parameters/authInQuery"},
+						{Ref: "#/components/parameters/authInHeader"},
+						{Ref: "#/components/parameters/csrf"},
+					},
+					RequestBody: nil, // TODO
+					Responses:   nil, // TODO
+				},
+			},
+			refPathWithID: {
+				Ref: "#/paths/~1path~1with~1{id}",
+			},
+		},
+		Components: &ogen.Components{
+			Schemas:   nil, // TODO
+			Responses: nil, // TODO
+			Parameters: map[string]ogen.Parameter{
+				authQ.Ident(): *authQ.Parameter,
+				authH.Ident(): *authH.Parameter,
+				csrf.Ident():  *csrf.Parameter,
+			},
```

Comment:
Maybe provide a cleaner API for declaring them?

```go
Parameters: ogen.NamedParamaters(authQ, authH, csrf),
```

Reply:
I think this is not necessary. This is only done here like this to ease construction I'd the expected result for the test. The end user will not construct the parameter map like this.